sidebars/docs.js

```diff
@@ -369,6 +384,26 @@ const sidebar = [
                 type: "doc",
                 id: "2.0/docs/patcher/guides/promotion-workflows",
               },
+              {
+                label: "Using Patcher Report",
+                type: "doc",
+                id: "2.0/docs/patcher/guides/report",
+              },
+              {
+                label: "Using Patcher Update",
+                type: "doc",
+                id: "2.0/docs/patcher/guides/update",
+              },
+              {
+                label: "Using Patcher Upgrade",
+                type: "doc",
+                id: "2.0/docs/patcher/guides/upgrade",
+              },
+              {
+                label: "Disable Telemetry",
```

Comment:
Disabling Telemetry?  I'm not a grammar expert, feels like a tense mismatch w/other guide titles